core/variant/variant.h

```diff
@@ -784,4 +784,13 @@ const Variant::ObjData &Variant::_get_obj() const {
 
 String vformat(const String &p_text, const Variant &p1 = Variant(), const Variant &p2 = Variant(), const Variant &p3 = Variant(), const Variant &p4 = Variant(), const Variant &p5 = Variant());
 
+template <typename... VarArgs>
+Callable Callable::bind(VarArgs... p_args) {
+	Variant args[sizeof...(p_args) + 1] = { p_args..., Variant() }; // +1 makes sure zero sized arrays are also supported.
```

Comment:
I like how this comment is copy-pasted to literally every vararg method xd

Reply:
If it's not broken don't fix it :P